src/main/java/jewellery/inventory/service/ResourceService.java

```diff
@@ -90,6 +99,49 @@ public List<ResourceQuantityResponseDto> getAllResourceQuantities() {
         .toList();
   }
 
+  public List<ResourceResponseDto> importResources(MultipartFile file) {
+    csvVerification(file);
+
+    List<ResourceRequestDto> resourcesDto = getImportedResources(file);
+
+    List<ResourceResponseDto> savedResources = new ArrayList<>();
+    for (ResourceRequestDto resourceRequestDto : resourcesDto) {
+      ResourceResponseDto savedResource = createResource(resourceRequestDto);
+      savedResources.add(savedResource);
+    }
+
+    return savedResources;
+  }
+
+  private void csvVerification(MultipartFile file) {
```

Comment:
Method names should be verbs
```suggestion
  private void verifyIsCsv(MultipartFile file) {
```